charts/minio-manager/README.md

```diff
@@ -30,13 +32,14 @@ The following table lists the parameters of this chart.
 |`minioAdminCredentials.secretKeys.useSsl`        | Name of the secret key for use ssl field                                   |MINIO_USE_SSL               |
 |`image.repository`                               | Repository of the minio image                                              |minio/mc                    |
 |`image.pullPolicy`                               | Kubernetes image pull policy                                               |IfNotPresent                |
-|`image.tag`                                      | Tag of the minio image                                                     |RELEASE.2021-01-05T05-03-58Z|
+|`image.tag`                                      | Tag of the minio image                                                     |RELEASE.2020-11-25T23-04-07Z|
```

Comment:
```suggestion
|`image.tag`                                      | Tag of the minio image. To avoid issues, this should match the MinIO server's version.      |RELEASE.2020-11-25T23-04-07Z|
```

Reply:
The fun part, it should not match the server version, it should be a compatible version, which are not built at the same time. Because YOLO!!!
From the chart we are currently using (v8.0.9), the server version is minio/minio:RELEASE.2020-12-03T05-49-24Z and the client is: minio/mc:RELEASE.2020-11-25T23-04-07Z